src/app/components/forms/Autocomplete.tsx

```diff
@@ -87,7 +71,14 @@ export const Autocomplete = forwardRef(function Autocomplete<
     }
   };
 
-  if (!showSuggestions || suggestions.length === 0) {
+  const shouldShowEmptyMessage =
+    showSuggestions && emptyMessage && suggestions.length === 0;
+
+  if (showSuggestions === false) {
```

Comment:
```suggestion
  if (!showSuggestions) {
```
A tester si ça fonctionne bien